### PR TITLE
Move daemonset from istio to serverless release

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Move daemonset from istio to serverless release

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/8646

### CODE REVIEW

File: resources/serverless/templates/daemonset-certs.yaml [hunk 1]
```diff
@@ -17,11 +17,12 @@ spec:
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
+      serviceAccountName: {{ template "fullname" . }}-controller-manager
```
Comment (michal-hudy) [inline resources/serverless/templates/daemonset-certs.yaml:20]: Use separate SA with proper scope

Comment (magicmatatjahu) [reply resources/serverless/templates/daemonset-certs.yaml]: Done :)

**Pull request merged**